cannot install 'botbuilder-applicationinsights' in qna-with-appinsights

In index.js file we have, const { ApplicationInsightsTelemetryClient, ApplicationInsightsWebserverMiddleware } = require('botbuilder-applicationinsights');

cannot Error: Cannot find module 'botbuilder-applicationinsights' and when i try to install it  - 'npm i botbuilder-applicationinsights' i get the below error

npm ERR! code E404
npm ERR! 404 Not Found: botbuilder-applicationinsights@latest

--- a/samples/javascript_nodejs/20.qna-with-appinsights/index.js
+++ b/samples/javascript_nodejs/20.qna-with-appinsights/index.js
@@ -4,7 +4,7 @@
 const path = require('path');
 const restify = require('restify');
 const { BotFrameworkAdapter } = require('botbuilder');
-const { ApplicationInsightsTelemetryClient, ApplicationInsightsWebserverMiddleware } = require('botbuilder-applicationinsights');
+const { ApplicationInsightsTelemetryClient, ApplicationInsightsWebserverMiddleware } = require('applicationinsights');
 const { BotConfiguration } = require('botframework-config');
 const { QnAMakerBot } = require('./bot');
 const { MyAppInsightsMiddleware } = require('./middleware');

